discern.lic

```diff
@@ -0,0 +1,46 @@
+custom_require.call(%w[common common-travel events spellmonitor drinfomon])
+
+class AutoDiscern
+    include DRC
+    include DRCA
+
+    def initialize()
+    settings = get_settings
+    if(settings.force_discerns)

```

Comment:
Is this a new setting? It feels more like it should be an argument to the script...

Reply:
If its a new setting, it needs to be added to base.yaml